digitaltwin_client/samples/digitaltwin_sample_device/digitaltwin_sample_device.c

```diff
@@ -62,9 +62,16 @@ static const int digitalTwinSampleDevice_mainPollingInterval = 1000;
 // Everytime the main loop wakes up, on the digitalTwinSampleDevice_sendTelemetryFrequency(th) pass will send a telemetry message 
 static const int digitalTwinSampleDevice_sendTelemetryFrequency = 20;
 
-// Name of the device capability model 
-#define DIGITALTWIN_SAMPLE_DEVICE_CAPABILITY_MODEL_URI "urn:azureiot:testdevicecapabilitymodel:1"
+//
+// TODO`s: Configure core settings of application for your Digital Twin
+//
+
+// TODO: Fill in DIGITALTWIN_SAMPLE_DEVICE_CAPABILITY_MODEL_ID. E.g. 
+#define DIGITALTWIN_SAMPLE_DEVICE_CAPABILITY_MODEL_ID "urn:YOUR_COMPANY_NAME_HERE:sample_device:1"
```

Comment:
This will break the PnP quickstart. We will need to update that and add a step to update this configuration.

Also the readme should say how to get a valid for this setting.

Reply:
Quickstart needs to change.